packages/create-neutrino-project/commands/init/templates/test/eslintrc.js

```diff
@@ -0,0 +1,5 @@
+const { Neutrino } = require('neutrino');
+
+const api = Neutrino();
```

Comment:
To save a declaration, let's export directly:

```js
module.exports = Neutrino().call('eslintrc');
```

Reply:
Let's also update the main Neutrino docs to suggest doing this too :-)